ynr/apps/api/v09/serializers.py

```diff
@@ -263,14 +272,14 @@ class Meta:
             "contact_details",
             "links",
             "memberships",
-            "image",
+            "images",
             "extra_fields",
             "thumbnail",
         )
 
     links = serializers.SerializerMethodField()
     other_names = OtherNameSerializer(many=True, read_only=True)
-    image = ImageSerializer(read_only=True)
+    images = serializers.SerializerMethodField()
```

Comment:
I'd have thought this would need changes to the tests?

Reply:
There was no test covering this so I have added one